src/indexers/chain/backing.ts

```diff
@@ -84,34 +112,59 @@ export const filter = $.filter(
   }
 );
 
+type BackingDiff = {
+  unback: Lovelace;
+  back: Lovelace;
+};
+
+type UnbackResult = {
+  projectId: Hex;
+  totalAmount: Lovelace;
+  actorAddress: Address;
+};
+
+type BackingTupleKey = [Hex, Address];
+
 export const event = $.event(
   async ({
     driver,
     connections: { sql, lucid, slotTimeInterpreter },
-    block: { slot },
+    block: { slot, time },
     tx,
     event: { indicies },
   }) => {
     const txTimeStart = slotTimeInterpreter.slotToAbsoluteTime(
       tx.body.validityInterval.invalidBefore ?? slot
     );
-    const unbackingMessage = extractCip20Message(tx)?.join("\n") || null;
-    // TODO: Bounded unbacked_at?
-    const unbackResult = await sql`
-      UPDATE chain.backing b
-      SET unbacking_message = ${unbackingMessage},
-          unbacked_at = ${txTimeStart}
-      FROM chain.output o
-      WHERE
-        o.id = b.id
-        AND (o.tx_id, o.tx_ix) IN ${sql(
-          tx.body.inputs.map((input) => sql([input.txId, input.index]))
-        )}
+    const message = extractCip20Message(tx)?.join("\n") || null;
+
+    const unbackResult = await sql<UnbackResult[]>`
+      SELECT
+        SUM(b.backing_amount) AS total_amount,
+        b.project_id,
+        b.backer_address AS actor_address
+      FROM chain.backing b
+      INNER JOIN chain.output o ON b.id = o.id
+      WHERE (o.tx_id, o.tx_ix) IN ${sql(
+        tx.body.inputs.map((input) => sql([input.txId, input.index]))
+      )}
+      GROUP BY
+        (b.project_id, b.backer_address)
     `;
+
+    // BackingTupleKey
+    const actionRef: Map<string, BackingDiff> = new Map<string, BackingDiff>();
+    unbackResult.forEach((value) => {
+      const key: BackingTupleKey = [value.projectId, value.actorAddress];
+      actionRef.set(toJson(key), {
```

Comment:
It's a throw-away key.

```suggestion
      const key = `${value.projectId}|${value.actorAddress}`;
      actionRef.set(key, {
```